IsTheMartin/mrtnmrls
# Updated code block

## Summary

Added `Code` astro component to display and highlight code block in pages

## Evidence

|Before|After|
|---|---|
|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/03f07039-d87a-46ee-8fcd-de476cbd4e1a)|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/2cb9a374-5bc5-4a28-a51e-ba9995699e91)|

## Concern Information

To test please follow the next steps:

1. Navigate to `/privacy`
2. Scroll down and observe code block in Attribution example

## Issue ticket

- resolves #88 

## Check list

- [ x ] I did add reviewers to this PR
